### ISSUE TITLE
Feature request: Multiple selection dropdown custom field type

### ISSUE THREAD

**Issue by huma2000** [wekan/wekan#4166]
Hi,

If possible, will be great having an extra custom field type that allows the user to select several options from a dropdown or option list.

Cheers

**Comment by mfilser** [comment]
This feature is already there, or did i something misunderstood?

![grafik](https://user-images.githubusercontent.com/13166201/142697683-738505ee-9e8c-4494-9c4e-3b8b131b6ccd.png)

**Comment by huma2000** [comment]
The dropdown allows only to select one of the values, the idea is having a dropdown with several options A, B, C ,D, E and being able to select A, C and D, not only A.
I hope I explained myself better now  hehehe

**Comment by mfilser** [comment]
Ah, yes, of course, i understand now.
Hm, another new custom field option like "Multi Select" or something else

**Comment by huma2000** [comment]
Exactly, basically as I am trying to use wekan to easy the management of the inpatients, having that option can allow me to create a quick list of the medications each animal is on, so we can see it easily
